dbt_server/server.py

```diff
@@ -13,7 +13,8 @@
 from dbt_server.exceptions import StateNotFoundException
 
 
-models.Base.metadata.create_all(bind=engine)
+models.Base.metadata.create_all(bind=engine, checkfirst=True)
```

Comment:
small heads up: it looks like `checkFirst` defaults to `True` in `create_all()`, so this might not resolve the table creation error you're seeing

Reply:
Yeeeaaaah I found that out the hard way. I don't know why this is failing, but I definitely haven't fixed it.